doc/developer-notes.md

```diff
@@ -71,11 +71,15 @@ required when doing so would need changes to significant pieces of existing
 code.
   - Variable (including function arguments) and namespace names are all lowercase, and may use `_` to
     separate words (snake_case).
-    - Class member variables have a `m_` prefix.
-    - Global variables have a `g_` prefix.
+    - Class member variables have a `m_` prefix, so it's obvious when variable
```

Comment:
WDYS to drop `m_` prefix if it's the public member variable of a class/struct?

Reply:
Maybe something to bring up in IRC or a separate issue. Personally I wouldn't mind it, as long as I could tell by sight when a member variable was being accessed (with `inst.var` or `this->var`, or `m_var` or whatever), and not confuse it for a local or global variable.